backend/sdk_adapter.go

```diff
@@ -0,0 +1,100 @@
+package backend
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/grafana/grafana-plugin-sdk-go/dataframe"
+	"github.com/grafana/grafana-plugin-sdk-go/genproto/pluginv2"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/expfmt"
+)
+
+// sdkAdapter adapter between protobuf and SDK interfaces.
+type sdkAdapter struct {
```

Comment:
Why does this both embed the two types, and also have them has named properties?